Cargo.toml

```diff
@@ -54,9 +54,6 @@ harness = false
 name = "06_rustup"
 
 [badges]
-travis-ci = { repository = "clap-rs/clap", branch = "master" }
-azure-devops = { project = "clap-rs/clap", pipeline = "clap-rs.clap" }
-coveralls = { repository = "clap-rs/clap", branch = "master" }
```

Comment:
Crates.io is removing support for badges

Reply:
I could find anything about it.

Reply:
[That](https://github.com/rust-lang/crates.io/issues/2436) is it, I guess.